### PR TITLE
Changed text entities to links in docs.

### PR DESCRIPTION
**Pull request by Revertron**
### Description of the Change

Right now structs, enums and etc. are mentioned by simply putting them in ``.
I changed them to actual links instead, so that documentation will be easier to browse.

### Benefits

Documentation is easier to browse.

### Possible Drawbacks 

None.

### CODE REVIEW

File: iroha_client_cli/src/main.rs [hunk 1]
```diff
@@ -280,7 +280,7 @@ mod account {
     /// Register account
     #[derive(StructOpt, Debug)]
     pub struct Register {
-        /// Id of account in form `name@domain_name'
+        /// Id of account in form `name@domain_name`
```
Comment (i1i1) [inline iroha_client_cli/src/main.rs:283]: This is actually intentional. This docs are used for cli help print

File: iroha_client_cli/src/main.rs [hunk 2]
```diff
@@ -406,7 +406,7 @@ mod asset {
     /// Register subcommand of asset
     #[derive(StructOpt, Debug)]
     pub struct Register {
-        /// Asset id for registering (in form of `name#domain_name')
+        /// Asset id for registering (in form of `name#domain_name`)
```
Comment (i1i1) [inline iroha_client_cli/src/main.rs:409]: Same for here

File: iroha_client_cli/src/main.rs [hunk 3]
```diff
@@ -438,10 +438,10 @@ mod asset {
...
     #[derive(StructOpt, Debug)]
     pub struct Mint {
-        /// Account id where asset is stored (in form of `name@domain_name')
+        /// Account id where asset is stored (in form of `name@domain_name`)
         #[structopt(long)]
         pub account: AccountId,
-        /// Asset id from which to mint (in form of `name#domain_name')
+        /// Asset id from which to mint (in form of `name#domain_name`)
         #[structopt(long)]
         pub asset: AssetDefinitionId,
```
Comment (i1i1) [inline iroha_client_cli/src/main.rs:446]: And in all other places :)

File: iroha_client_cli/src/main.rs (hunk 1 above)
Comment (Revertron) [reply iroha_client_cli/src/main.rs:283]: Only in `main.rs`?

File: iroha_client_cli/src/main.rs (hunk 2 above)
Comment (Revertron) [reply iroha_client_cli/src/main.rs]: Fixed.

File: iroha_client_cli/src/main.rs (hunk 3 above)
Comment (Revertron) [reply iroha_client_cli/src/main.rs]: Fixed.

**Review by i1i1 — DISMISSED** [review]

**Review by eadventurous — DISMISSED** [review]

**Review by i1i1 — APPROVED** [review]

**Pull request merged**